src/main/java/org/spongepowered/common/mixin/core/item/MixinItemBlock.java

```diff
@@ -35,7 +36,7 @@
 import java.util.Optional;
 
 @NonnullByDefault
-@Mixin(net.minecraft.item.ItemBlock.class)
+@Mixin(value = { net.minecraft.item.ItemBlock.class, net.minecraft.item.ItemAir.class })
```

Comment:
*sigh*

ItemAir doesn't extend ItemBlock does it...

I guess we'll keep the name instead but a comment about why ItemAir is used here should be put.